### DISCUSSION TITLE
My application got rejected

### DISCUSSION THREAD

**Discussion by Sazid99246** [community/community#52944] [GitHub Education] [answered]
Hi. i have registered at Santa Fe college and they provided me an edu mail like this: **student_id@sfcollege.edu**. This is an edu mail, right? But when I wanted to join the github student developer pack with this email, it asked me to provide the photo of the id card. But I know that if I have edu mail, it automatically enrolls me in the student developer pack. Why does it want a date issued id card or other documents? Then I wanted to upload a picture of the id card, but it doesn't work, why are these happening to me?

**Comment by Aditya753b** [comment, marked as answer]
Yeah, actually it asks for ID proof whether u are studying there or not. the best way that u get ur application get accepted is by applying on phone and taking pictures from the phone when it asked for ID proof. There may be a chance for rejection if u upload an already-taken photo.Just take a photo at the photo proof section.

**Reply by eniolaseun1** [reply]
I have done everything, still not working

**Reply by Sazid99246** [reply]
I am also getting errors again and again while going to upload the student id.

**Reply by eniolaseun1** [reply]
This is way too annoying

**Reply by Aditya753b** [reply]
Well post a new request cuz this one is closed,to be Frank even I'm also facing some issues

**Comment by eniolaseun1** [comment]
I think this is the part I say goodbye to github

**Reply by Aditya753b** [reply]
nah bro i just got it finally if u dont mind im free to help u personally just mention me here : )

**Reply by anmolpal-github** [reply]
how can this be sorted. got rejected as i don't have student email id from my college

**Reply by Aditya753b** [reply]
is the email id is given to staff only or student too?

**Reply by innomaxx** [reply]
Same. Lots of attempts with all the combinations - still not working. Futhermore my rear camera isn't working on their approval page. Great implementation, GitHub, perfect code 👍🏻

**Reply by Mamatata2** [reply]
Hello. Can you help me? Instagram: death_by_hand Telegram: bober_vasyaa

**Comment by anmolpal-github** [comment]
They don’t give emails to anyone.

**Reply by Aditya753b** [reply]
can u keep a screenshot of what type of mail they are asking at email page?

**Comment by Suman-Mukherje** [comment]
> nah bro i just got it finally if u dont mind im free to help u personally just mention me here : )

Please help me for getting this as i am getting rejected 8 times till now , i uploaded my id image very clearly, gave all correct information!!

**Reply by Aditya753b** [reply]
hey hi please keep a photo with blurring ur email which states the reason of rejection.

**Reply by Suman-Mukherje** [reply]
![Inkedgithub message](https://github.com/community/community/assets/69708181/a9b6412c-7714-466e-a373-24111f98f7ce)

**Reply by Aditya753b** [reply]
the reason of ur rejection is related with ur mail id. genrally u need to give .edu mail to verify not @gmail one's. If u want .edu mail or the official mail id of the college ask ur staff by stating ur reason.

**Reply by Sharifu-Msuya** [reply]
is that so?
